### ISSUE TITLE
Cannot send notification with cc or bcc

### ISSUE THREAD

**Issue by JeanLucEsser** [verbb/formie#927]
**Description**
When using the fields cc or bcc, I have this error in the logs

```
2022-06-03 09:18:39 [ERROR] Notification email  ^`^|Contact (client) ^`^} could not be sent for submission  ^`^|6566 ^`^}. Error: Cannot assign array to property verbb\formie\elements\SentNotification::$cc of type ?string /srv/app/generationopera/htdocs/vendor/verbb/formie/src/services/SentNotifications.php:80
2022-06-03 09:18:39 [ERROR] Email payload: {"charset":"utf-8","from":{"contact@generationopera.fr":"G  n  ration Op  ra"},"replyTo":{"xxx@generationxxx.fr":""},"to":{"xxx@gmail.com":""},"cc":{"xxx@generationxxx.fr":""},"bcc":"","subject":"Votre demande    G  n  ration Op  ra","body":"<p>Bonjour</p>"}
```

<img width="1057" alt="image" src="https://user-images.githubusercontent.com/14908712/171805886-85778436-b34a-49b7-9f29-c9c50d438ef1.png">

**Additional info**
- Plugin version: beta 10
- Craft version: 4.0.3

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.0-beta.10",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.0-beta.11](https://github.com/verbb/formie/releases/tag/2.0.0-beta.11)
